src-python/drbench/drbench/drbench.py

```diff
@@ -62,7 +56,7 @@ def mhp(ctx, bench, fork, nprocs, sycl_cpu, sycl_gpu):
     if sycl_gpu:
         # mhp-bench will spread GPUs over ranks automatically, so no
         # pinning is needed
-        env = 'ONEAPI_DEVICE_SELECTOR=level_zero:gpu'
+        env = 'ONEAPI_DEVICE_SELECTOR=\'level_zero:gpu;ext_oneapi_cuda:gpu\''
```

Comment:
It works as expected on our CPU-only and CUDA systems. I could not test on devcloud because CI is consuming my only slot., but CI would fail so that must be OK, too.